projects/plugins/jetpack/modules/masterbar/inline-help/class-inline-help.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Inline Help
+ *
+ * Handles providing a LiveChat icon within WPAdmin until such time
+ * as the full live chat experience can be run in a non-Calypso environment.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+use Automattic\Jetpack\Redirect;
+use Automattic\Jetpack\Status;
+
+/**
+ * Class Inline_Help.
+ */
+class Inline_Help {
+
+	/**
+	 * Inline_Help constructor.
+	 */
+	public function __construct() {
+		$this->register_actions();
+	}
+
+	/**
+	 * Registers actions
+	 *
+	 * @return void
+	 */
+	public function register_actions() {
+
+		// Do not inject the FAB icon on embedded screens since the parent window may already contain a FAB icon.
```

Comment:
Should we insert the icon at all in the block editor?

Reply:
I did this because that matches up with what happens in Calypo Gutenframe. There is a FAB on Calypso editor.